### PR TITLE
Fix inline create routes in conformity with fetch ones

### PR DESCRIPTION
**Pull request by pxpm**
## WHY

There was a bug in `FetchOperation` that we are fixing now in #3995 . This operation suffers from the same evil. 

### BEFORE - What was wrong? What was happening before this PR?

Inline create routes would not respect the the `kebabing` of the entity done in `fetch_and_create` and would not match neither the fetch operation routes or the inline modal ones, so developer needed to manually define `entity => smt` in more cases than needed.

### AFTER - What is happening after this PR?

Reduced the cases for when specifying `entity => smt` is needed, probably only needed now when adding Inline create to multiple relations that usually is setup as singular crud (eg: `Route::crud('icon')), and then have a `belongsToMany` (eg: `icons` as a relation name).

### How did you achieve that, in technical terms?

Applied the same thing as in #3995 when setting up the routes

### Is it a breaking change or non-breaking change?

It's possible a breaking one for some edge cases, not for the "simpler" ones.

### How can we test the before & after?

??

### CODE REVIEW

**Comment by tabacitu** [comment]
@pxpm the way I read it, this PR brings a breaking change for everybody who's manually defined the route for `inline_create`, correct? If so, let's postpone this into v12. We have enough breaking changes already - the upgrade guide already counts _21 steps_ 😱

If it were a non-breaking change like #3995 I'd merge it, but it doesn't look like it.

File: src/app/Http/Controllers/Operations/InlineCreateOperation.php [hunk 1]
```diff
@@ -16,13 +17,14 @@ trait InlineCreateOperation
      */
     protected function setupInlineCreateRoutes($segment, $routeName, $controller)
     {
-        Route::post($segment.'/inline/create/modal', [
-            'as'        => $segment.'-inline-create',
+        $inlineRouteName = Str::kebab(str_replace('_', '-', $routeName));
+        Route::post($segment.'/inline-create/modal', [
```
Comment (tabacitu) [inline src/app/Http/Controllers/Operations/InlineCreateOperation.php:21]:
```suggestion
        Route::post($segment.'/inline/create/modal', [
```

File: src/app/Http/Controllers/Operations/InlineCreateOperation.php [hunk 2]
```diff
@@ -16,13 +17,14 @@ trait InlineCreateOperation
...
+            'as'        => $inlineRouteName.'-inline-create',
             'uses'      => $controller.'@getInlineCreateModal',
             'operation' => 'InlineCreate',
         ]);
-        Route::post($segment.'/inline/create', [
-            'as'        => $segment.'-inline-create-save',
+        Route::post($segment.'/inline-create', [
```
Comment (tabacitu) [inline src/app/Http/Controllers/Operations/InlineCreateOperation.php:26]:
```suggestion
        Route::post($segment.'/inline/create', [
```

**Comment by pxpm** [comment]
I applied your commits, it still is a breaking change for people that fixed the behaviour. Sad.

File: src/app/Http/Controllers/Operations/InlineCreateOperation.php [hunk 3]
```diff
@@ -16,13 +17,14 @@ trait InlineCreateOperation
      */
     protected function setupInlineCreateRoutes($segment, $routeName, $controller)
     {
+        $inlineRouteName = Str::kebab(str_replace('_', '-', $routeName));
```
Comment (tabacitu) [inline src/app/Http/Controllers/Operations/InlineCreateOperation.php:20]:
```suggestion
        $segment = Str::kebab(str_replace('_', '-', $routeName));
```

**Comment by tabacitu** [comment]
This will be affected by [the open-core split](https://github.com/Laravel-Backpack/CRUD/issues/4098). But we can't merge it right now, since we can't add any more breaking changes to 4.2. We'll have to manually re-do this PR towards `backpack/pro` later, I'm afraid. Let's leave this open until we do.
